api/src/search/backend/load_agencies_to_index.py

```diff
@@ -73,14 +74,27 @@ def fetch_agencies(self) -> Sequence[Agency]:
 
     def load_agencies(self, agencies: Sequence[Agency]) -> None:
         logger.info("Loading agencies...")
+        active_agency_subquery = (
+            _construct_active_inner_query(Agency.agency_id)
+            .union(_construct_active_inner_query(Agency.top_level_agency_id))
+            .subquery()
+        )
+
+        agency_id_stmt = select(active_agency_subquery).distinct()
+        active_agencies = set(self.db_session.execute(agency_id_stmt).unique().scalars().all())
```

Comment:
Does just this work?
```suggestion
        active_agencies = set(self.db_session.execute(agency_id_stmt).scalars())
```

Reply:
Works!